src/tool-header.scss

```diff
@@ -0,0 +1,295 @@
+
+    .#{$block}__title {
+      @include fd-reset-paddings-x();
+
+      @media (max-width: $fd-tool-header-screen-size-sm) {
+        display: none;
+      }
+    }
+  }
+
+  .#{$block}__icon {
+    color: var(--fdTool_Header_Icon_Color_Default);
+    font-size: 1rem;
+    line-height: 1;
+
+    &--inverted {
+      color: var(--fdTool_Header_Icon_Color_Inverted_Default);
+      background: var(--fdTool_Header_Icon_Background_Color_Inverted_Default);
+    }
+
+    @each $set-name, $set-params in $fd-tool-header-icon-semantic-values {
+      &--#{$set-name} {
+        color: map_get($set-params, 'iconColor');
+
+        &.#{$block}__icon--inverted {
+          color: map_get($set-params, 'iconColorInverted');
+          background: map_get($set-params, 'bgColorInverted');
+        }
+      }
+    }
+  }
+
+  .#{$block}__object-status {
+    @each $set-name, $set-params in $fd-tool-header-object-status-semantic-values {
+      &--#{$set-name} {
+        color: map_get($set-params, 'color');
+        text-shadow: none;
+
+        [class*="sap-icon"] {
+          color: map_get($set-params, 'color');
+        }
+
+        &[class*="inverted"] {
+          text-shadow: none;
+          color: map_get($set-params, 'iconColorInverted');
+
+          [class*="icon"] {
+            color: map_get($set-params, 'iconColorInverted');
+          }
+
+          background: map_get($set-params, 'bgColorInverted');
+
+          @include fd-hover () {
+            color: map_get($set-params, 'iconColorInverted');
+
+            [class*="icon"] {
+              color: map_get($set-params, 'iconColorInverted');
+            }
+
+            background: map_get($set-params, 'bgColorInverted');
+          }
+
+          &[class*="link"] {
+            @include fd-hover () {
+              color: map_get($set-params, 'iconColorInverted');
+
+              [class*="icon"] {
+                color: map_get($set-params, 'iconColorInverted');
+              }
+
+              background: map_get($set-params, 'bgColorInverted');
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .#{$block}__input-group {
+    background: var(--sapShellColor);
+    border: 0.0625rem solid var(--sapShell_InteractiveBorderColor);
+
+    input {
+      color: var(--sapShell_TextColor);
+
+      &::placeholder {
+        color: var(--sapShell_InteractiveTextColor);
+      }
+    }
+
+    @include fd-hover() {
+      background: var(--sapShell_Hover_Background);
+      border-color: var(--sapShell_InteractiveBorderColor);
+    }
+  }
+
+  // MODIFIER CLASSES
+  &--menu {
+    @include fd-set-paddings-x-equal(0.5rem);
+  }
+
+  @each $set-name, $set-padding in $fd-tool-header-responsive-paddings {
+    &--#{$set-name} {
+      padding: map_get($set-padding, 'padding');
+    }
+  }
+
+  &--responsive-paddings {
+    @media (max-width: $fd-tool-header-screen-size-sm) {
+      @include fd-set-paddings-x-equal(1rem);
+    }
+
+    @media (min-width: 600px) and (max-width: 1023px) {
```

Comment:
Can we have these responsive breakpoints moved into some global variables for all styles, so we keep breakpoints as consistent as possible?